modules/instrument_builder/js/instrument_builder.instrument.js

```diff
@@ -120,10 +120,15 @@ var Instrument = {
 
                         // Add dropdown and special naming when no date format is set
                         // (i.e when addDateElement() is used)
-                        if (element.Options.dateFormat === "") {
-                            elName = elName + "_date";
-                            dropdown = "select{@}" + elName + "_status" +
+                        if (!element.Options.dateFormat) {
+                          if (!element.Description) {
+                            // Prevents undefined being added to linst file.
+                            element.Description = '';
+                          }
```

Comment:
similarly to making $elname='', i don't think the description should ever be empty or undefined because it's a required field

Reply:
@zaliqarosli I'll test again I forgot to with last changes. I'm unsure why I thought this was needed and what you're writing makes sense.